freedomofpress/securedrop-client
# test: run in parallel across 4 workers, 30s -> 12-14s

# Description

this PR makes the change to run the test suite in parallel. We should also monitor https://github.com/pytest-dev/pytest-xdist/issues/229 because right now one cannot run tests in random order as well as run in parallel (:cry:), currently favoring running tests fast but open to discussion

# Test Plan

0. run tests
1. observe it takes 12-14 seconds

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [x] These changes should not need testing in Qubes (test only)
